asset/scss/responsive.scss

```diff
@@ -0,0 +1,817 @@
+		.ti-write{
+			font-size: 30px;
+			margin-right: 10px;
+		}
+		.ti-calendar{
+			font-size: 30px;
+			margin-right: 10px;
+		}
+	}
+	.content-1{
+		text-align: center;
+		font-size: 14px;
+		font-style: italic;
+		line-height: 24px;
+	}
+	.content-2{
+		font-size: 13px;	
+		line-height: 21px;
+		margin-top: 31px;
+	}
+	.dai_img{
+		margin-top: 33px;
+	}
+	.button-daily{
+		text-align: center;
+		margin-top: 40px;
+		margin-bottom: 50px;
+	}
+}
+
+.rectangle-3{
+	.our-rectangle3{
+		.rec-group{
+			width: 0px;
+			padding-left: 15px;
+		}
+	}
+}
+
+.member{
+ 	.member-head{
+		font-size: 36px;
+		margin-top: 50px;
+		text-align: center;
+		.ti-user{
+			font-size: 30px;
+			margin-right: 12px;
+		}
+ 	}
+ 	.content-1{
+ 		text-align: center;
+		font-size: 14px;
+		line-height: 24px;
+	}
+	.member-content{
+		line-height: 21px;
+		font-size: 13px;
+		.member-detail-1{
+			margin-top: 57px;
+		}
+		.member-detail-2{
+			margin-top: 20px;	
+		}
+	}
+	.img{
+		text-align: center;
+		margin-top: -270px;
+	}
+	.button-member{
+		text-align: center;
+		margin-top: 30px;
+		margin-bottom: 38px;
+	}
+}
+
+.customer{
+	.customer-head{
+		font-size: 36px;
+		margin-top: 50px;
+		text-align: center;
+		.ti-face-smile{
+			font-size: 30px;
+			margin-right: 12px;
+		}
+	}
+	.content-1{
+		text-align: center;
+		font-size: 14px;
+		line-height: 24px;
+	}
+	.img{
+		margin-top:-222px;
+	}
+	.customer-content{
+		line-height: 21px;
+		font-size: 13px;
+		.customer-detail-1{
+			margin-top: 26px;
+		}
+		.customer-detail-2{
+			margin-top: 17px;	
+			.ti-cup{
+				color: #8BC34A;
+			}
+		}
+	}
+	.button-customer{
+		text-align: center;
+		margin-bottom: 50px;
+  	margin-top: 35px;
+	}
+}
+
+.rectangle4 {
+  height: 544px;
+  .head-rec{  
+  font-size: 36px;  
+  height: 48px;
+	}
+	.owl-carousel{
+		.owl-stage-outer {
+  		height: 155px;
+		}
+	}
+	.our-customers {
+  	height: 127px;
+  	margin-top: 24px;
+  	.cus-name{
+  		line-height: 19px;
+  	}
+	}
+	.ti-quote-left{
+		font-size: 64px;
+		margin-top: 50px;
+		padding-left: 0px;
+	}
+	.tes-text{
+  	font-size: 17px;
+  	padding-left: 0px;
+  	margin-top: 110px;
+	}
+  .ti-quote-right{
+		font-size: 64px;
+		margin-top: -25px;
+		text-align: center;
+		padding-left: 0px;
+	}
+}
+
+footer{
+	.row{
+		margin-top: 54px;
+		.col-3{
+			margin-top: 43px;
+			ul{
+				li{
+					margin-left: -3px;
+					float: left;
+					.ti-twitter-alt{
+						margin-left: 18px;
+						}
+					.ti-youtube{
+						margin-left: 18px;
+						}
+					.ti-linkedin{
+						margin-left: 18px;
+					}
+				}
+			}
+		}
+		.copyright{
+			margin-top: -70px;
+		}
+	}
+}
+
+//end media
+}
+
+@media screen and (max-width: 767px){
+.mb-view{
+	display: block;
+}
+
+.pc-view{
+	display: none;
+}
+
+body{
+	width: 100%;
+	margin: 0px;
+	padding:0px;
+}
+
+.start-now{
+	height: 40px;
+	width: 144px;
+	font-size: 14px;
+}
+
+header{
+	height: 300px;
+	.container-head{
+		margin-top: 15px;
+		.sign-in{
+			margin-right: 15px;
+			height: 30px;
```

Comment:
nut sign in sao phai set height width co dinh vay

Reply:
@anhht-hapo k set height width thì e định làm thế nào???